addons/project/views/project_project_views.xml

```diff
@@ -427,88 +413,82 @@
-                                                <div>
-                                                    <span class="o_value"><t t-esc="record.open_task_count.value"/></span>
-                                                    <span class="o_label ms-1"><t t-esc="record.label_tasks.value"/></span>
-                                                </div>
-                                            </a>
-                                            <a groups='project.group_project_milestone' t-if="record.allow_milestones and record.allow_milestones.raw_value and record.milestone_count.raw_value &gt; 0"
-                                                class="o_kanban_inline_block btn-link text-dark small"
-                                                role="button"
-                                                name="action_get_list_view"
-                                                type="object"
-                                                t-attf-title="#{record.milestone_count_reached.value} Milestones reached out of #{record.milestone_count.value}"
-                                            >
-                                                <span class="fa fa-flag me-1"/>
-                                                <t t-out="record.milestone_count_reached.value"/>/<t t-out="record.milestone_count.value"/>
-                                            </a>
-                                        </div>
-                                        <field name="activity_ids" widget="kanban_activity"/>
+                                    <div t-if="record.alias_email.value" class="text-muted text-truncate" t-att-title="record.alias_email.value">
+                                        <span class="fa fa-envelope-o me-2" aria-label="Domain Alias" title="Domain Alias"></span><field name="alias_email"/>
                                     </div>
-                                    <div class="oe_kanban_bottom_right">
-                                        <field name="user_id" widget="many2one_avatar_user" domain="[('share', '=', False)]"/>
-                                        <field t-if="record.last_update_status.value &amp;&amp; widget.editable" name="last_update_status" widget="project_state_selection"/>
-                                        <span t-if="record.last_update_status.value &amp;&amp; !widget.editable" t-att-class="'o_status_bubble mx-0 o_color_bubble_' + record.last_update_color.value" t-att-title="record.last_update_status.value"></span>
+                                    <div t-if="record.rating_active.raw_value and record.rating_count.raw_value &gt; 0" class="d-flex text-muted" groups="project.group_project_rating">
+                                        <b class="me-1">
+                                            <span style="font-weight:bold;" class="fa mt4 fa-smile-o text-success" t-if="record.rating_avg.raw_value &gt;= 3.66" title="Average Rating: Satisfied" role="img" aria-label="Happy face"/>
```

Comment:
remove style and use tag `<strong>` or bootstrap class instead

Reply:
not working strong tag or bootstrap class